tests/packagedcode/data/m2/bytebuddy/pom.xml.package.json

```diff
@@ -30,7 +30,14 @@
   "vcs_url": "git+https://github.com/raphw/byte-buddy.git", 
   "copyright": null, 
   "license_expression": null, 
-  "declared_license": "The Apache Software License, Version 2.0\nhttp://www.apache.org/licenses/LICENSE-2.0.txt\nA business-friendly OSS license", 
+  "declared_license": [
+    {
+      "name": "The Apache Software License, Version 2.0", 
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.txt", 
+      "comments": "A business-friendly OSS license", 
+      "distribution": "A business-friendly OSS license"
```

Comment:
Distribution should be repo (but I am not sure we care for that field at all)
Here is the original XML POM snippet:
```
    <licenses>
        <license>
            <name>The Apache Software License, Version 2.0</name>
            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
            <distribution>repo</distribution>
            <comments>A business-friendly OSS license</comments>
        </license>
</licenses>
```

Reply:
Add a 'text" key in ordered dictionary to keep the original license text info.